### PR TITLE
v1.0.2

### PR DESCRIPTION
**Pull request by nolanmar511**
Release notes: https://github.com/googleapis/cloud-profiler-nodejs/releases/edit/untagged-da8128b01d9fd6f4ca32

### CODE REVIEW

**Comment by ofrobots** [comment]
Some nits:

> There are two user-visible changes.

Most users will read this as 'breaking changes'. I would suggest rewording this as 'There are two enhancements'.

Can you convert `knative` to a link.

Otherwise looks good.

**Comment by kalyanac** [comment]
lgtm.

**Comment by nolanmar511** [comment]
Changes to release notes are done. I made [knative](https://cloud.google.com/knative/)  a link, and switched to suggested wording.
